src/components/Common/Button/button.css.ts

```diff
@@ -18,4 +18,10 @@ export const buttonStyle = style({
       color: vars.color.grey_6,
     },
   },
-});
\ No newline at end of file
+});
+
+export const outlinedStyle = style({
+  background: vars.color.white,
+  border: `1px solid ${vars.color.grey_4}`,
```

Comment:
한 가지 팁입니다!!
테두리를 border로 줄 경우 border를 포함한 전체 요소 너비가 달라지기 때문에 이로 인해 내부 요소나, 인접한 다른 요소에 레이아웃 시프트가 발생할 수 있어요. 같은 이슈로 오픈소스에 PR을 오픈한 내용이 있는데요, PR에 있는 AS-IS, TO-BE를 참고하면 이해가 쉬울 것 같습니다!!
https://github.com/kubeflow/notebooks/pull/561

저는 이 문제를 해결하기 위해 box shadow: inset을 주로 활용하고 있어요. 이 친구는 요소 사이즈를 유지하면서 내부에 테두리를 만들어서 레이아웃 시프트 이슈를 방지할 수 있습니다.

```suggestion
boxShadow: `inset 0 0 0 1px ${vars.color.grey_4}`,
```

Reply:
앗 이전에 학습했던 내용인데 익숙한 걸로 해버렸네요ㅜ.ㅜ 리마인드 감자합니다🥔